packages/ui/src/components/ResourceBlock/ResourceBlock.tsx

```diff
@@ -0,0 +1,196 @@
+import ExclamationCircleIcon from "@heroicons/react/20/solid/ExclamationCircleIcon";
+import ArrowRightIcon from "@heroicons/react/20/solid/ArrowRightIcon";
+
+import { HeadingLevel } from "../Heading";
+import { headingStyles } from "../Heading/styles";
+import Link, { LinkProps } from "../Link";
+import { HydrogenAttributes } from "../../types";
+
+export const colorOptions = [
+  "primary",
+  "secondary",
+  "tertiary",
+  "quaternary",
+  "quinary",
+] as const;
+
+type CardColor = (typeof colorOptions)[number];
+
+const headingBarStyleMap: Record<CardColor, Record<string, string>> = {
+  primary: {
+    "data-h2-background-color": "base(primary.lightest)",
+    "data-h2-border-bottom": "base(primary.darkest)",
+  },
+  secondary: {
+    "data-h2-background-color": "base(secondary.lightest)",
+    "data-h2-border-bottom": "base(secondary.darkest)",
+  },
+  tertiary: {
+    "data-h2-background-color": "base(tertiary.lightest)",
+    "data-h2-border-bottom": "base(tertiary.darkest)",
+  },
+  quaternary: {
+    "data-h2-background-color": "base(quaternary.lightest)",
+    "data-h2-border-bottom": "base(quaternary.darkest)",
+  },
+  quinary: {
+    "data-h2-background-color": "base(quinary.lightest)",
+    "data-h2-border-bottom": "base(quinary.darkest)",
+  },
+};
+
+const wrapperStyleMap: Record<CardColor, Record<string, string>> = {
+  primary: {
+    "data-h2-color": "base(primary.darkest)",
+  },
+  secondary: {
+    "data-h2-color": "base(secondary.darkest)",
+  },
+  tertiary: {
+    "data-h2-color": "base(tertiary.darkest)",
+  },
+  quaternary: {
+    "data-h2-color": "base(quaternary.darkest)",
+  },
+  quinary: {
+    "data-h2-color": "base(quinary.darkest)",
+  },
+};
+
+interface RootProps {
+  title: ReactNode;
+  headingColor?: CardColor;
+  headingAs?: HeadingLevel;
+  children: ReactElement<ItemProps> | Array<ReactElement<ItemProps>>; // Restricts children to only expected items;
+}
+
+const Root = ({
+  title,
+  headingColor = "primary",
+  headingAs = "h3",
+  children,
+}: RootProps) => {
+  // prepare heading text element
+  const HeadingTextElement = headingAs;
+  const commonHeadingTextStyles = { ...headingStyles["h4"] };
+  delete commonHeadingTextStyles["data-h2-margin"];
+  return (
+    <div
+      data-h2-shadow="base(larger)"
+      data-h2-border-radius="base(rounded)"
+      data-h2-background-color="base(foreground)"
+    >
+      {/* heading bar */}
+      <div
+        {...headingBarStyleMap[headingColor]}
+        data-h2-border-radius="base(rounded rounded 0 0)"
+        data-h2-border-bottom="base(1px solid)"
+        data-h2-padding="base(x1) p-tablet(x1 x1.5  )"
+        data-h2-text-align="base(center)"
+      >
+        {/* wrapper */}
+        <div {...wrapperStyleMap[headingColor]}>
+          <HeadingTextElement
+            {...commonHeadingTextStyles}
+            data-h2-text-align="base(center) p-tablet(left)"
+          >
+            {title}
+          </HeadingTextElement>
```

Comment:
Oh actually you can use the <Heading /> component instead. That way you don't need to import the styles and delete the margin. It would look something like:

```tsx
<Heading
  level={headingAs}
  size="h4"
  data-h2-margin="base(0)"
  data-h2-text-align="base(center) p-tablet(left)"
>
  {title}
</Heading>
```

Reply:
Oh yeah, that's way better!
[use heading component](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11200/commits/0a7247dfa09fdbbf0cdbefd61e0a9626b74b704b)